djautotask/tests/fixture_utils.py

```diff
@@ -0,0 +1,42 @@
+from suds.client import Client
+from atws.wrapper import QueryCursor
+import urllib
+import os
+
+
+def mock_query_generator(suds_objects):
+    """
+    A generator to return suds objects to mock the same behaviour as the
+    ATWS Wrapper query method.
+    """
+    for obj in suds_objects:
+        yield obj
+
+
+def set_attributes(suds_object, fixture_object):
+    """
+    Set attributes on suds object from the given fixture.
+    """
+    for key, value in fixture_object.items():
+        setattr(suds_object, key, value)
+
+    return suds_object
+
+
+def generate_objects(object_type, fixture_objects):
+    """
+    Generate multiple objects based on the given fixtures.
+    """
+    path = os.path.abspath("djautotask/tests/atws.wsdl")
```

Comment:
I think just a quick explanaition in a comment of what this atws.wsdl is, and why we are using it would be good here for posterity.